docs/faq.md

```diff
@@ -23,10 +23,31 @@ toc_depth: 4
 
 # Frequently Asked Questions (FAQ)
 
-## `<Foo>Timeout` error in the logs
+## A local OS error occurred while connecting to a device
+???+ faq "A local OS error occurred while connecting to a device"
+
+    When running ANTA, you can receive `A local OS error occurred while connecting to <device>` errors. The underlying [`OSError`](https://docs.python.org/3/library/exceptions.html#OSError) exception can have various reasons: `[Errno 24] Too many open files` or `[Errno 16] Device or resource busy`.
+
+    This usually means that the operating system refused to open a new file descriptor (or socket) for the ANTA process. This might be due to the hard limit for open file descriptors currently set for the ANTA process.
```

Comment:
is it really the hard limit?

Reply:
That's why I said "might".
There are many reasons why the OS can refuse to open a file descriptor.